src/unittests/conversion.spec.ts

```diff
@@ -285,3 +286,25 @@ g.test('pack2x16float')
       `pack2x16float(${inputs}) returned [${got_str}]. Expected [${expect}]`
     );
   });
+
+g.test('pack2x16unorm')
+  .paramsSimple([
+    // Normals
+    { inputs: [0, 0], result: 0x00000000 },
+    { inputs: [1, 0], result: 0x0000ffff },
+    { inputs: [1, 1], result: 0xffffffff },
```

Comment:
Please can we have a case for exceeds `1`?

Reply:
Done